src/tracing/node_trace_writer.cc

```diff
@@ -110,8 +110,8 @@ void NodeTraceWriter::FlushSignalCb(uv_async_t* signal) {
   trace_writer->FlushPrivate();
 }
 
-// TODO: Remove (is it necessary to change the API? Since because of WriteSuffix
-// it no longer matters whether it's true or false)
+// TODO(matthewloring): Remove (is it necessary to change the API?
```

Comment:
Nope, that one should stay in to remind me to clean up some of this. Thanks for checking!